jenkins-integration/beaker/install/shared/45_install_hiera_config.rb

```diff
@@ -0,0 +1,39 @@
+require 'puppet/gatling/config'
+require 'tmpdir'
+require 'yaml'
+
+test_name 'Install hiera config'
+
+def target_hiera_config_path(host)
+  return on(host, puppet('config print hiera_config')).stdout.chomp
+end
+
+def get_source_hiera_config(host, hiera_config)
+  source_hiera_config_path = hiera_config[:source_file]
+  if source_hiera_config_path.nil? || source_hiera_config_path.empty?
+    source_hiera_config_path = target_hiera_config_path(host)
```

Comment:
It just seems pretty heavy-handed to me that, if I'm reading this code correctly, you're *always* going to overwrite the hiera.config file from the packages, even if the job doesn't specify any values for *either* of the two hiera settings.  It feels like there might be tests where we wanted to be 100% sure we were testing the default configuration and this seems like it makes it less obvious / certain that we will be.

In a vacuum I'd probably rather keep it simpler, but, I'm eager to keep moving forward, so I'll just merge this as-is for now.